examples/gdk-pixbuf.toml

```diff
@@ -0,0 +1,13 @@
+description = "Update gdk-pixbuf cache"
+
+[check]
+paths = [
+    "/usr/lib/gdk-pixbuf-2.0/2.10.0/loaders"
```

Comment:
Where does this path come from? Is it likely to change in future versions of gdk-pixbuf or any of the packages involved in this process?

If so you should change it to a path that uses wildcards like in `ghc-pkg.toml`

Reply:
The path to use is the one I did provide [here](https://dev.getsol.us/D13280#228282)

Reply:
No I get that. I see in the diff that that's the path that the files are being placed in. My point is that that path looks dependent on the version of something else and I wonder if another package being updated might cause the path that files get put into to move around thus breaking this usysconf trigger. In that case the path in this trigger should use a wildcard so that it doesn't break on package updates. 

Perhaps that path is a known constant and this concern is unfounded. I'm unfamiliar with any of the packages involved or what logic is used to determine that path.

Reply:
It does appear to be a constant value, at least for the last 4 years on Solus. Gdk-PixBuf is currently at version 2.42.8 in Solus's repos, however it still uses that 2.10 path.

Reply:
The version has stayed the same for at least 5 years (hardcoded in their [meson.build](https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/master/meson.build#L25) ever since it has existed). So it seems at least very long-lived. But as I'm also not too familiar with gdk-pixbuf I don't know how strong the guarantee of it staying the same is.